velox/expression/tests/ExpressionRunnerTest.cpp

```diff
@@ -57,13 +58,46 @@ DEFINE_string(
     "simplified paths.\n"
     "common: evaluate the expression using common path and print out results.\n"
     "simplified: evaluate the expression using simplified path and print out "
-    "results.");
+    "results.\n"
+    "query: evaluate SQL query specified in --sql or --sql_path and print out "
+    "results. If --input_path is specified, the query may reference it as "
+    "table 't'.");
+
+static bool validateMode(const char* flagName, const std::string& value) {
+  static const std::unordered_set<std::string> kModes = {
+      "common", "simplified", "verify", "query"};
+  if (kModes.count(value) != 1) {
+    std::cout << "Invalid value for --" << flagName << ": " << value << ". ";
+    std::cout << "Valid values are: ";
+    bool first = true;
+    for (const auto& mode : kModes) {
+      if (!first) {
+        std::cout << ", ";
+      } else {
+        first = false;
+      }
+      std::cout << mode;
+    }
+    std::cout << "." << std::endl;
+    return false;
+  }
+
+  return true;
+}
+
+DEFINE_validator(mode, &validateMode);
 
 DEFINE_int32(
     num_rows,
     10,
     "Maximum number of rows to process. Zero means 'all rows'. Applies to "
-    "'common' and 'simplified' modes only. Ignord for 'verify' mode.");
+    "'common' and 'simplified' modes only. Ignored for 'verify' mode.");
+
+DEFINE_string(
```

Comment:
Consider add velox_expression_runner_xxx? dittos

Reply:
You suggest to add velox_expression_runner_ prefix to the gflags? I think this will make the flags too long and inconvenient to type.